static/js/components/email/EmailCompositionDialog.js

```diff
@@ -78,6 +79,9 @@ export default class EmailCompositionDialog extends React.Component {
   constructor (props: EmailDialogProps) {
     super(props);
     this.state = editorStateFromProps(props);
+    if (R.isNil(this.state.editorState)) {
```

Comment:
In that case, the last function in the `editorStateFromProps` composition needs to be modified. Right now it looks like this:

```js
S.maybe({}, R.objOf('editorState'))
```

The `S.maybe` function has a the type signature `maybe :: b -> (a -> b) -> Maybe a -> b` (or something along those lines). Basically, the arguments are `maybe(defaultValue, onJustFunc, Maybe)`. If the `Maybe` is `Nothing`, the function will return `defaultValue`, if the `Maybe` is `Just`, it will return `onJustFunc(value)`, where `value` is the value wrapped up in the `Just`.

So right now if the function in question receives a `Nothing` it will return `{}`, and if it receives a `Just(x)`, it will return `R.objOf('editorState', x)`, which returns `{editorState: x}`. So I think all we would need to change is the default value, which should go from being `{}` to being `{editorState: EditorState.createEmpty()}`. So that would look like:

```js
maybe({ editorState: EditorState.createEmpty() }, R.objOf('editorState'))
```

 Because `EditorState` is an Immutable value we shouldn't need to worry about creating it every time the `editorStateFromProps` function is called (I think).